src/DETHRACE/common/car.c

```diff
@@ -5679,7 +5679,7 @@ int CollideCameraWithOtherCars(br_vector3* car_pos, br_vector3* cam_pos) {
                 BrVector3Add(&pos_car_space, &p, &dir);
                 BrVector3SetFloat(&tv, 0.03f, 0.03f, 0.03f);
                 BrVector3Sub(&bnds.min, &c->bounds[0].min, &tv);
-                BrVector3Add(&bnds.max, &c->bounds[0].min, &tv);
+                BrVector3Add(&bnds.max, &c->bounds[0].max, &tv);
```

Comment:
nice!

Reply:
I introduced the bug in c6010a2466995737bb3de7fc6b400b33b5140cf7 in this pr, so nothing special :)